v3/src/components/graph/components/background.tsx

```diff
@@ -58,18 +56,19 @@ export const Background = forwardRef<SVGGElement, IProps>((props, ref) => {
     previousMarqueeRect = useRef<rTreeRect>(),
 
     onDragStart = useCallback((event:{ x: number; y: number; sourceEvent: {shiftKey: boolean} }) => {
+      const plotBounds = layout.computedBounds.get('plot') as Bounds
```

Comment:
```suggestion
      const { computedBounds } = layout,
      plotBounds = computedBounds.get('plot') as Bounds
```
and update the dependencies to just `layout`.